docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -23,18 +23,18 @@ This section is a work in progress and gathers the philosophy under which extens
 
 === Why an extension framework
 
-Quarkus’s mission is to transform your full application and the libraries it uses into an optimal output for GraalVM.
+Quarkus' mission is to transform your entire application including the libraries it uses, into an artifact optimized for GraalVM.
```

Comment:
I had it as `Quarkus’` initially and @LightGuard corrected it to `Quarkus’s`
Let's not change it

Reply:
Done